app/views/movies/_movie_show_screening.html.erb

```diff
@@ -5,5 +5,5 @@
   <span class="screening-link"><%= link_to "Add / Manage Screenings", movie_screenings_path(@movie), id: "add_screening_link_movies_partial" %></span>
 <% else %> <!-- #if movie has been seen -->
   <h2>You Haven't Seen this One Yet!
-  <span class="screening-link"><%= link_to "Mark as Watched", screenings_path(:screening => { notes: nil }, tmdb_id: @movie.tmdb_id, from: "movies_index", page: params[:page]), :method => :post, id: "mark_watched_link_movies_partial", remote: true %></span></h2>
-<% end %> <!-- #if movie has been seen -->
\ No newline at end of file
+  <span class="screening-link"><%= link_to "Mark as Watched", screenings_path(screening: { location_watched: current_user.default_location }, tmdb_id: @movie.tmdb_id, from: "movies_index", page: params[:page]), :method => :post, id: "mark_watched_link_movies_partial", remote: true %></span></h2>
```

Comment:
We don't have Rubocop, but while yer in here, mind cleaning this up a bit with something like:

```erb
<% else %> <!-- #if movie has been seen -->
  <h2>You Haven't Seen this One Yet!
    <span class="screening-link">
      <%= link_to "Mark as Watched",
        screenings_path(
          screening: { location_watched: current_user.default_location },
          tmdb_id: @movie.tmdb_id,
          from: "movies_index",
          page: params[:page]
        ),
        method: :post,
        id: "mark_watched_link_movies_partial",
        remote: true %>
    </span>
  </h2>
```